lib/usbmux/usbmux.js

```diff
@@ -0,0 +1,137 @@
+import net from 'net';
+import _ from 'lodash';
+import B from 'bluebird';
+import plist from '../util/plist.js';
+import LengthBasedSplitter from '../util/trasformer/length-based-splitter';
+import UsbmuxDecoder from './transformer/usbmux-decoder.js';
+import UsbmuxEncoder from './transformer/usbmux-encoder.js';
+
+const DEFAULT_USBMUXD_SOCKET = '/var/run/usbmuxd';
+const PROG_NAME = 'appium-ios-device';
+const CLIENT_VERSION_STRING = 'appium-ios-device-0.0.1';
```

Comment:
version number should be taken from package.json

Reply:
It is just a placeholder. It has no meaning. I can also remove 0.0.1 version

Reply:
yes, it would be better to remove the version number otherwise